ukat/mapping/t2star.py

```diff
@@ -262,21 +276,33 @@ def to_nifti(self, output_directory=os.getcwd(), base_file_name='Output',
             Eg., base_file_name = 'Output' will result in 'Output.nii.gz'.
         maps : list or 'all', optional
             List of maps to save to NIFTI. This should either the string "all"
-            or a list of maps from ["t2star", "m0", "r2star", "mask"].
+            or a list of maps from ["t2star", "t2star_err", "m0",
+            "m0_err", "r2star", "mask"].
         """
         os.makedirs(output_directory, exist_ok=True)
         base_path = os.path.join(output_directory, base_file_name)
         if maps == 'all' or maps == ['all']:
             maps = ['t2star', 'm0', 'r2star', 'mask']
+            if self.method == '2p_exp':
+                maps += ['t2star_err', 'm0_err']
         if isinstance(maps, list):
             for result in maps:
                 if result == 't2star' or result == 't2star_map':
                     t2star_nifti = nib.Nifti1Image(self.t2star_map,
                                                    affine=self.affine)
                     nib.save(t2star_nifti, base_path + '_t2star_map.nii.gz')
+                elif result == 't2star_err' or result == 't2star_err_map':

```

Comment:
Maybe you could add `and self.method == '2p_exp'`? The idea is to avoid problems in the case that `self.method == loglin` and the user accidentally includes 't2star_err' and/or 'm0_err' in the maps list.

Reply:
Personally I think its more confusing if someone explicitly asks for x maps but the function only outputs y maps. It doesn't fail to export if `method == loglin`, it just exports zeros. Maybe a warning is a compromise?

Reply:
Good point, I think a warning is the right thing to do. I'll merge after that.